network/server/tchannelthrift/node/service.go

```diff
@@ -332,9 +341,16 @@ func (s *service) FetchBlocksMetadataRaw(tctx thrift.Context, req *rpc.FetchBloc
 		includeChecksums = *req.IncludeChecksums
 	}
 
-	fetched, nextPageToken, err := s.db.FetchBlocksMetadata(
-		ctx, s.idPool.GetBinaryID(ctx, req.NameSpace), uint32(req.Shard),
-		start, end, req.Limit, pageToken, includeSizes, includeChecksums)
+	buffer := s.bytesPool.Get(len(req.NameSpace))
```

Comment:
The main thing is that `ts.BinaryID(...)` now requires a `checked.Bytes` ref to be passed.  The issue here is that there's no easy way to reuse the `checked.Bytes` from the bytes pool without using the actual backing bytes.

Perhaps I'll add a simple `sync.Pool` for the use of `checked.Bytes` in this file.  Let me update with that.